runner/process_linux.go

```
@@ -4,38 +4,61 @@ package runner
 
 import (
 	"syscall"
+
+	"golang.org/x/sys/unix"
 )
 
 const syscallStopSignal = syscall.Signal(syscall.SIGTRAP | 0x80)
```

golangci-lint (2.12.2)

[error] unnecessary conversion

(unconvert)